core/src/log.cpp

```diff
@@ -263,4 +268,20 @@ const char* Logger::streamFormat(memory::Stream& stream, const char* format, con
     }
 
     return format;
+}
+
+std::string cubos::core::logSystemError()
+{
+#ifdef _WIN32
+    static char message[256] = {0};
+    FormatMessage(FORMAT_MESSAGE_FROM_SYSTEM | FORMAT_MESSAGE_IGNORE_INSERTS, 0, WSAGetLastError(), 0, message, 256, 0);
```

Comment:
`WSAGetLastError` is specific to `winsock2`, so it doesn't make sense after all for this to be in `log.hpp` :pensive: maybe just roll back the change then? You can also just add this utility in a private header in `core/src/net/log.hpp` or smth.

Reply:
Can I add a `utils.hpp` for stuff like `SOCKET` and this function? Or how should I organize this

Reply:
Sure yeah, you can add a `net/utils.hpp`.